<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QDialog" name="Client">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QPushButton" name="ConB">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>40</y>
     <width>80</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Подключение</string>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenB">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Открыть</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SendB">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Отправить</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>160</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="ServHorizontalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="ServL">
      <property name="text">
       <string>Сервер</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ServLI"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>29</x>
     <y>70</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="PortHorizontalLayout">
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="PortL">
      <property name="text">
       <string>   Порт </string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="PortLI"/>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="StatusLI">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
